requirements/edx/kernel.in

```diff
@@ -159,4 +159,4 @@ webob
 web-fragments                       # Provides the ability to render fragments of web pages
 XBlock[django]                      # Courseware component architecture
 xss-utils                           # https://github.com/edx/edx-platform/pull/20633 Fix XSS via Translations
-slumber
+slumber                             # The following dependency is unsupported and used by the DeprecatedRestApiClient
```

Comment:
This should be added in alphabetical order. Unfortunately the dependencies starting with `s` aren't all in order, so you could just put this after `six`, and/or reorder the rest of the `s` dependencies in a separate commit.